src/sentry/models/file.py

```diff
@@ -442,7 +442,7 @@ class FileBlobIndex(Model):
     __core__ = False
 
     file = FlexibleForeignKey("sentry.File")
-    blob = FlexibleForeignKey("sentry.FileBlob")
+    blob = FlexibleForeignKey("sentry.FileBlob", on_delete=models.PROTECT)
```

Comment:
You'll need to generate a migration for this, it'll probably just be a no-op to update the state.

Reply:
Hrm, django wants to change the schema of the table which I don't think is a good idea given the size of the table. 

```diff
+    operations = [
+        migrations.AlterField(
+            model_name='fileblobindex',
+            name='blob',
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(on_delete=django.db.models.deletion.PROTECT, to='sentry.FileBlob'),
+        ),
+    ]
```

Was the migration that was generated.

Reply:
Nevermind. After looking at the generated SQL it would have been a no-op change so I can make this a state only migration.